micromamba/tests/test_create.py

```diff
@@ -661,6 +666,89 @@ def test_create_envs_dirs(tmp_root_prefix: Path, tmp_path: Path):
     assert (tmp_path / env_name / "conda-meta" / "history").exists()
 
 
+@pytest.mark.parametrize("set_in_conda_envs_dirs", (False, True))
+@pytest.mark.parametrize("set_in_condarc", (False, True))
+@pytest.mark.parametrize("cli_root_prefix", (False, True))
+@pytest.mark.parametrize("check_config", (False, True))
+def test_root_prefix_precedence(
+    tmp_path,
+    tmp_home,
+    monkeypatch,
+    set_in_condarc,
+    set_in_conda_envs_dirs,
+    cli_root_prefix,
+    check_config,
```

Comment:
To be slightly more explicit and precise
```suggestion
    check_config_only,
```